src/main/java/net/glowstone/Explosion.java

```diff
@@ -255,7 +260,6 @@ private void setBlockOnFire(GlowBlock block) {
             entity.damage(damage, source, damageCause);
 
             if (entity instanceof GlowPlayer) {
```

Comment:
Will do.
Actually, you pointed another possible error with explosions out.
Flying players will, at least in paper 1.10, take damage from explosions.
However, this will need more testing -> another pr.